### ISSUE TITLE
how about 200 classes?

### ISSUE THREAD

**Issue by xinsuinizhuan** [AlexeyAB/darknet#8256]
my custom data, 200 classes，i train them 24 hours, but the loss don't recude:
![图片](https://user-images.githubusercontent.com/40679769/143676439-d16a8efd-59fc-4cfb-8261-d29e744be720.png)
 and the avg loss is nan
![图片](https://user-images.githubusercontent.com/40679769/143676463-e4140b6d-b50a-496a-a3bc-73b558355370.png)
 how shoul i do?
![图片](https://user-images.githubusercontent.com/40679769/143676566-72386c59-cb0b-416b-a6c7-f9ef3a457c41.png)
then i only change the classes and filters

**Comment by lsd1994** [comment]
If you got avg loss = nan, you should stop training because there are some mistakes in you date, cfg, etc.
